### DISCUSSION TITLE
Numbers changed

### DISCUSSION THREAD

**Discussion by TekyTech** [sumatrapdfreader/sumatrapdf#4717] [General]
Someone sent a bunch of financial statements in PDFs. Upon opening in Sumatra, numbers have changed! Both on screen and when printed. When same documents are opened with Adobe, the numbers show as originally intended by the sender. 
Any ideas?

**Comment by GitHubRulesOK** [comment]
Hmm has been known for MUPDF to do that in the past (same as a classic case of  IBM scanners with OCR font numeric errors)

If they are private send to the developer @kjk for analysis to see if there is a clear reason.
But do mention in message which version & variant exhibits such problems. 
You could also try MuPDF to see if that is the root cause and if so open a bugzilla report there.
